pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -172,6 +172,33 @@ Once complete, continue to [attach your cluster][attach-cluster] to Kommander.
 
 You should expect to see that cert-manager will fail to deploy due to your existing `cert-manager` installation. This is expected and can be ignored.
 
+### Kommander Cluster with custom SSL certificate
+
+When attaching a cluster, it is expected that the managed cluster was deploying apps as federated by the management cluster.
+If the management cluster was initialised using a custom SSL certificate, the managed cluster may fail cloning the managers’ service repository. You may need to check the status of the federated GIT repository resource to see the error;
+
+```
+kubectl get gitrepo -A --kubeconfig MANAGED-KUBECONFIG
+[..]
+unable to clone 'https://MANAGER_INGRESS_ADDRESS/dkp/kommander/git/kommander/kommander': Get "https://MANAGER_INGRESS_ADDRESS/dkp/kommander/git/kommander/kommander/info/refs?service=git-upload-pack": x509: certificate signed by unknown authority
+[..]
+```
+
+The deployment fails because the managed cluster uses the wrong CA certificate to verify access to the management clusters’ GIT repository. For working around this issue we need to patch the `gitserver-ca` secret within the `kommander-flux` namespace on the managed cluster with the CA certificate as stored in the `kommander-traefik-certificate` secret within the `kommander`namespace on the management cluster.
```

Comment:
```suggestion
The deployment fails because the managed cluster uses the wrong CA certificate to verify access to the management clusters’ git repository. For working around this issue you patch the `gitserver-ca` secret within the `kommander-flux` namespace on the managed cluster with the CA certificate stored in the `kommander-traefik-certificate` secret within the `kommander`namespace on the management cluster.
```

Reply:
I didn't know how to quote your edit @makkes, so I just edited your edit to include another change. #inception.

Reply:
@gvasquezvargas I think you solved it perfectly, much appreciated